content/en/docs/quickstart.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Quickstart: Sign and validate a container image"
+description: "Quickly set up a OCI-based registry and use notation to sign and validate a container image"
+type: docs
+weight: 2
+---
+
+## Prerequisites
+
+Before you begin, you need:
+
+* [Docker desktop](https://www.docker.com/products/docker-desktop/) installed and running.
```

Comment:
@dtzar @SteveLasker @zr-msft Is it necessary to specify Docker Desktop as a prerequisite? Maybe we just need to require users to install Docker.

Reply:
fixed